compiler/include/WhileStmt.h

```diff
@@ -33,7 +33,7 @@ class WhileStmt : public LoopStmt
 protected:
   WhileStmt(Expr*      sym, BlockStmt* initBody);
   WhileStmt(VarSymbol* sym, BlockStmt* initBody);
-  virtual ~WhileStmt();
+  ~WhileStmt() override;
```

Comment:
It would be nice to align the W in the destructor below the W in the constructor, i.e. shift this line one space to the left.